disnake/channel.py

```diff
@@ -1362,6 +1362,9 @@ async def create_instance(
             The stage instance's topic.
         privacy_level: :class:`StagePrivacyLevel`
             The stage instance's privacy level. Defaults to :attr:`StagePrivacyLevel.guild_only`.
+
+            .. deprecated:: 2.4
+
```

Comment:
I might be wrong, but when I said that changing the documentation is needed, I thought about a note mentioning that only `guild_only` is supported.

Setting it to public works too, so I guess it's not that important, a deprecation warning if someone tries to set it to public would be nice.

Reply:
I agree, the property itself doesn't seem to be deprecated, only the `PUBLIC` value is

Reply:
A deprecation warning (with `utils.warn_deprecated`) when someone tries to use `StagePrivacyLevel.public` might be good too.